python/paddle/fluid/layers/nn.py

```diff
@@ -9936,8 +9918,8 @@ def slice(input, axes, starts, ends):
             'ends': ends,
             'infer_flags': infer_flags
         }
-        outs = core.ops.slice(inputs, attrs)
-        return outs['Out'][0]
+        return core.ops.slice(input, 'axes', axes, 'starts', starts, 'ends',
```

Comment:
Move https://github.com/PaddlePaddle/Paddle/pull/23181/files#diff-f8fc1afdc1f06d4c47bc0933d53bf3c6R9915 in static graph mode.

Reply:
Done, thanks.